pkg/app/api/grpcapi/api.go

```diff
@@ -437,7 +440,20 @@ func (a *API) GetPlanPreviewResults(ctx context.Context, req *apiservice.GetPlan
 		return nil, err
 	}
 
-	const freshDuration = 24 * time.Hour
+	const (
+		freshDuration               = 24 * time.Hour
+		defaultCommandHandleTimeout = 5 * time.Minute
+	)
+
+	var (
+		handledCommands = make([]string, 0, len(req.Commands))
+		results         = make([]*model.PlanPreviewCommandResult, 0, len(req.Commands))
+	)
+
+	commandHandleTimeout := time.Duration(req.CommandHandleTimeout) * time.Second
```

Comment:
To be more clear, I feel like we'd better mention this is the timeout in seconds.

`pkg/app/api/service/apiservice/service.protoL124`

```proto
message GetPlanPreviewResultsRequest {
    repeated string commands = 1;
    // timeout in second
    int64 command_handle_timeout = 2;
}
```

Reply:
Sure.